homeassistant/components/modbus/modbus.py

```diff
@@ -322,6 +322,10 @@ async def async_pymodbus_call(self, unit, address, value, use_call):
         if self._config_delay:
             return None
         async with self._lock:
-            return await self.hass.async_add_executor_job(
+            result = await self.hass.async_add_executor_job(
                 self._pymodbus_call, unit, address, value, use_call
             )
+            if self._config_type == "serial":
+                # small delay until next request/response
+                await asyncio.sleep(30 / 1000)
```

Comment:
Is there a reason we don't use `0.03`?

Reply:
Documentation, this is easy to read as 30 ms, whereas with 0.03 you have to calculate. Since this is 2 constants, there are no difference in the runtime (pyc file contains 0.03)